### ISSUE TITLE
Appleworks CWK FIle Example

### ISSUE THREAD

**Issue by brycenesbitt** [teacurran/appleworks-parser#1]
Would you accept contributions of additional AppleWorks files?  I've got some that don't import into libreOffice.

**Comment by teacurran** [comment]
yeah, if you have some I'd be happy to take a look. I don't spend much time on this project though so I can't promise anything.
